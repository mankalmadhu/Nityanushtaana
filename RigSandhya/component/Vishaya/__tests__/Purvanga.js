import 'react-native';
import React from 'react';
import ಪೂರ್ವಾ೦ಗ from '../Purvanga';

import Renderer from 'react-test-renderer';

test("ಶೀರ್ಷಿಕೆವಿಷಯ" , () => {

expect(Renderer.create(
    <ಪೂರ್ವಾ೦ಗ >
      Some Text
    </ಪೂರ್ವಾ೦ಗ>
).toJSON()).toMatchSnapshot();

});