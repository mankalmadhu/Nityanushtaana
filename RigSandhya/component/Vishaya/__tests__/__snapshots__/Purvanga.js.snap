// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಶೀರ್ಷಿಕೆವಿಷಯ 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#6695e2",
      "fontFamily": "Ganapati",
      "fontSize": 18,
      "lineHeight": 50,
      "marginLeft": 12,
      "textAlign": "justify",
    }
  }
>
  Some Text
</Text>
`;
